Update migrate-ad-fs-application-overview.md (#248)
MicrosoftDocs/entra-docs: docs/identity/enterprise-apps/migrate-ad-fs-application-overview.md

Application readiness is evaluated based on following predefined AD FS application configuration tests. The tests are run automatically and the results are displayed in the AD FS application migration dashboard as a **Migration status**. If the AD FS configuration isn't compatible with a Microsoft Entra configuration, you get specific guidance on how to address the configuration in Microsoft Entra ID.

## AD FS application migration insights status updates

> Once there is an update to the application, internal agents syncs the updates within few minutes. However, AD FS migration insights jobs responsible for evaluating the updates and computes new migration status. Those jobs are scheduled to run every 24 hours, which means data will be computed only once in a day at around 00:00 UTC timezone.

Reviewer: ```suggestion
When the application is updated, internal agents sync the updates within few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).
```
Edits for Microsoft style and grammar

> When the application is updated, internal agents sync the updates within few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).

Reviewer: ```suggestion
When the application is updated, internal agents sync the updates within a few minutes. However, AD FS migration insights jobs are responsible for evaluating the updates and compute a new migration status. Those jobs are scheduled to run every 24 hours, which means that the data will be computed only once in a day, at around 00:00 Coordinated Universal Time (UTC).
```
